### ISSUE TITLE
Update code standards to ES6/7

### ISSUE THREAD

**Issue by karmacoma** [LiskHQ/lisk#519]
Corresponding with the planned updates to the code standards in `lisk-js`, `lisk-explorer`. We should also update the code standards from ES5 to ES6/7 in the `lisk`. Initially using eslint which is already in-place.

**Comment by andyhu** [comment]
Can we also ditch caolan/async for built-in async / await + bluebird and switch to webpack instead of grunt? It's even better if it can switch from mocha to Jest!

**Comment by pietrovismara** [comment]
I'd like to contribute, is any help needed on this or other issues?

**Comment by karmacoma** [comment]
@pietrovismara Thanks for the offer of support, but we will take care of the first phase of migration, which is this issue.

**Comment by karmacoma** [comment]
> Can we also ditch caolan/async for built-in async / await + bluebird and switch to webpack instead of grunt? It's even better if it can switch from mocha to Jest!

@andyhu: Yes we will, once this initial migration is complete, we start refactoring more parts of the code.

**Comment by karmacoma** [comment]
Closing in favour of #672
